sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -162,7 +162,10 @@ static void initializeIntegrationsAndProcessors(
       options.setGestureTargetLocators(gestureTargetLocators);
     }
     options.setMainThreadChecker(AndroidMainThreadChecker.getInstance());
-    options.setMemoryCollector(new AndroidMemoryCollector());
+    if (options.getCollectors().isEmpty()) {
```

Comment:
`l`: just a thought - since this a list, should we just add to it no matter what? Even if the users added their own collectors, we could still add ours? 

Not applicable for `JavaMemoryCollector` though, since it'd be duplicating the values of `AndroidMemoryCollector` kinda

Reply:
Actually, only the last added Collectors would work, so we'd overwrite user's defined collectors.
Maybe we could set them as first in the list, using something like `collectors.add(0, collector)`
However, at the moment the collector is marked `@Internal`. 
Should we cover this case anyway?